integration/cypress/integration/base/header.spec.ts

```diff
@@ -19,7 +19,7 @@ context("Header", () => {
   it("navigates to the machine list when clicking on the logo as a non admin", () => {
     cy.get(".p-navigation__logo .p-navigation__link").click();
     cy.location("pathname").should("eq", generateNewURL("/machines"));
-    cy.get(".p-navigation__link.is-selected a").should("not.exist");
+    cy.get(".p-navigation__link.is-selected a").contains("Machines");
```

Comment:
I'm not sure how this ever worked before? If the url is at "/machines" then the highlighted nav link has to be "Machines"